tasks/grunt-stylelint.js

```diff
@@ -13,23 +13,23 @@ module.exports = function ( grunt ) {
 		options.files = this.filesSrc.filter( function ( file ) {
 			return grunt.file.isFile( file );
 		} );
+		options.formatter = 'string';
 
 		styleLint.lint( options ).then( function ( data ) {
 			data.results.forEach( function ( result ) {
 				if ( !result.errored ) {
```

Comment:
The built-in ["verbose"](https://github.com/stylelint/stylelint/blob/master/src/formatters/verboseFormatter.js) formatter [display](https://cloud.githubusercontent.com/assets/808227/14970073/0ebc8bd4-10be-11e6-9398-4aafa0657fa4.png) a file-by-file status.